src/app/accounts/update-account-ip.ts

```diff
@@ -17,28 +23,28 @@ export const updateAccountIPsInfo = async ({
   logger: Logger
 }): Promise<void | RepositoryError> =>
   asyncRunInSpan(
-    "app.users.updateAccountIPsInfo",
+    "app.accounts.updateAccountIPsInfo",
```

Comment:
this wrap is not necessary because in sessionContext we are using `Accounts.updateAccountIPsInfo` which is already a wrapper